public/resources/ts/calculate-player-stats.ts

```diff
@@ -1,5 +1,10 @@
 import * as helper from "../../../common/helper.js";
 import { STATS_BONUS } from "../../../common/constants.js";
+import { REFORGES } from "../../../src/constants/reforges.js";
```

Comment:
instead of importing everything it would be better to then just use `import * as CONTANTS from "../../../common/constants.js"`